src/ImageService.Business/Commands/ImageNews/GetImageNewsCommand.cs

```diff
@@ -1,45 +1,45 @@
-﻿using LT.DigitalOffice.ImageService.Business.Commands.ImageNews.Interfaces;
+﻿using System;
+using LT.DigitalOffice.ImageService.Business.Commands.ImageNews.Interfaces;
 using LT.DigitalOffice.ImageService.Data.Interfaces;
 using LT.DigitalOffice.ImageService.Mappers.Responses.Interfaces;
 using LT.DigitalOffice.ImageService.Models.Db;
 using LT.DigitalOffice.ImageService.Models.Dto.Responses;
 using LT.DigitalOffice.Kernel.Enums;
 using LT.DigitalOffice.Kernel.Responses;
-using System;
 
 namespace LT.DigitalOffice.ImageService.Business.Commands.ImageNews
 {
-    public class GetImageNewsCommand : IGetImageNewsCommand
-    {
-        private readonly IImageNewsRepository _imageNewsRepository;
-        private readonly IImageDataResponseMapper _imageNewsResponseMapper;
+  public class GetImageNewsCommand : IGetImageNewsCommand
+  {
+    private readonly IImageNewsRepository _imageNewsRepository;
+    private readonly IImageDataResponseMapper _imageNewsResponseMapper;
 
-        public GetImageNewsCommand(
-            IImageNewsRepository imageNewsRepository,
-            IImageDataResponseMapper imageNewsResponseMapper)
-        {
-            _imageNewsRepository = imageNewsRepository;
-            _imageNewsResponseMapper = imageNewsResponseMapper;
-        }
+    public GetImageNewsCommand(
+        IImageNewsRepository imageNewsRepository,
+        IImageDataResponseMapper imageNewsResponseMapper)
+    {
+      _imageNewsRepository = imageNewsRepository;
+      _imageNewsResponseMapper = imageNewsResponseMapper;
+    }
 
-        public OperationResultResponse<ImageDataResponse> Execute(Guid imageId)
-        {
-            OperationResultResponse<ImageDataResponse> response = new();
+    public OperationResultResponse<ImageDataResponse> Execute(Guid imageId)
+    {
+      OperationResultResponse<ImageDataResponse> response = new();
 
-            DbImageNews dbImageNews = _imageNewsRepository.Get(imageId);
+      DbImageNews dbImageNews = _imageNewsRepository.Get(imageId);
```

Comment:
response.Body = _imageNewsRepository.Get(imageId);
What for create extra variable?

Reply:
We use the created variable 2 times, I don't see anything wrong here

Reply:
done